src/migrations/migration.rs

```diff
@@ -126,13 +132,30 @@ async fn _read_names(dir: &Path) -> anyhow::Result<Vec<PathBuf>> {
         {
             continue;
         }
+
+        if !has_old_filename && old_name_re.is_match(item.file_name().to_str().unwrap()) {
+            has_old_filename = true;
+        }
+
         result.push(item.path());
     }
+
+    if has_old_filename {
```

Comment:
Maybe we should be checking for mixed old and new format and print a warning that something is severely wrong.

Or even error since we are probably in a middle of a rebase.